documentation/docs/.vuepress/config.js

```diff
@@ -0,0 +1,19 @@
+import { defaultTheme } from "@vuepress/theme-default";
+import { defineUserConfig } from "vuepress";
+
+export default defineUserConfig({
+  base: "/documentation/",
+  lang: "en-US",
+  title: "Documentation",
+  description: "Developer Documentation",
+  theme: defaultTheme({
+    logo: "/images/BCID_H_rgb_pos.png",
```

Comment:
This image is not actually part of the repo, please either add the image or remove the logo